### PR TITLE
optimize xpns_organize_panes

### PR DESCRIPTION
**Pull request by ReDTerraN**
Instead of calling global ${TMUX_XPANES_EXEC} multiple times, we combine the layout commands into a single local $layout_command string variable and execute it in a single call to ${TMUX_XPANES_EXEC} at the end of the script.

This can should save some overhead and improve readability.

Since ${_args_num} is an integer, it is more efficient to use arithmetic evaluation with (( )) instead of the [[ ]] test operator. So I have changed it to (( _args_num > 1 )).

Please have a look at the commit and make sure its working as intended.

Cheers

### CODE REVIEW

**Comment by ReDTerraN** [comment]
Looks like its not working as expected, I will look into this later this week.

File: bin/xpanes [hunk 1]
```diff
@@ -774,20 +774,19 @@ xpns_organize_panes() {
...
-  if [[ "${_args_num}" -eq 1 ]]; then
-    ${TMUX_XPANES_EXEC} select-layout -t "${_window_name}" even-horizontal
-  elif [[ "${_args_num}" -gt 1 ]]; then
-    ${TMUX_XPANES_EXEC} select-layout -t "${_window_name}" tiled
+  local layout_command="select-layout -t ${_window_name} even-horizontal"
+  if [[ "${_args_num}" -gt 1 ]]; then
+    layout_command="select-layout -t ${_window_name} tiled"
   fi
-  ## ----------------
+s
```
Comment (greymd) [inline bin/xpanes]: Is this `s` typo, isn't it?

```suggestion
```

Comment (ReDTerraN) [reply bin/xpanes]: image

Correct, I added it by mistake to the code while changing back to test operator

**Comment by ReDTerraN** [comment]
I found the issue, me preventing globbing and word splitting with double quotation marks made my input not work correctly.
As this is the exception that SC2086 talk about of what we are doing here, (building command line) and combined we are safe that this output will not be contaminated, I have disabled the check here.

**Comment by greymd** [comment]
Approved

**Pull request merged**